cmd/lakectl/cmd/fs.go

```diff
@@ -159,6 +159,36 @@ var fsUploadCmd = &cobra.Command{
 	},
 }
 
+var fsStageCmd = &cobra.Command{
+	Use:    "stage <path uri>",
+	Short:  "stage an object at the specified URI",
```

Comment:
Please explain better what this command does: adds a reference / imports an existing object, and allows it be managed in lakeFS.